src/warnet/graph.py

```diff
@@ -180,6 +184,7 @@ def inquirer_create_network(project_path: Path):
         custom_network_path,
         fork_observer,
         fork_observer_query_interval,
+        fork_observer, # This enables caddy whenever fork-observer is enabled
```

Comment:
Hmmm, now that logging is enabled by default, perhaps caddy must also be enabled by default in this PR @m3dwards ?

Reply:
I'd say yes? if the dashboard is meant to be how a user interacts with logs , then logs: enabled implies caddy:enabled in my mind.

Reply:
Now I'm thinking about it I think the logic should be if logging or if fork observer then yes caddy. I'm not sure we need the ability to disable caddy.

Reply:
And technically, logging isn't enabled by default. It's only enabled if metrics or logs are switched to on in your network definition.

Reply:
mmm ok. I see now in https://github.com/bitcoin-dev-project/warnet/pull/565/commits/0c4f76c624826c408918f0859b463a4502a106dc which I need to remove also (didn't spot in the rebase)

Reply:
See: https://github.com/bitcoin-dev-project/warnet/blob/main/src/warnet/deploy.py#L56